src/app/ApplicationTemplate.Shared/Presentation/Authentication/CreateAccountFormViewModel.cs

```diff
@@ -20,19 +44,138 @@ public string Email
 			set => this.Set(value);
 		}
 
+		public string PhoneNumber
+		{
+			get => this.Get<string>();
+			set => this.Set(value);
+		}
+
+		public string SecondaryPhoneNumber
+		{
+			get => this.Get<string>();
+			set => this.Set(value);
+		}
+
+		public string PostalCode
+		{
+			get => this.Get<string>();
+			set => this.Set(value);
+		}
+
+		public DateTimeOffset DateOfBirth
+		{
+			get => this.Get<DateTimeOffset>(initialValue: DateTimeOffset.Now);
+			set => this.Set(value);
+		}
+
 		public string Password
 		{
 			get => this.Get<string>();
 			set => this.Set(value);
 		}
+
+		public string ConfirmPassword
+		{
+			get => this.Get<string>();
+			set => this.Set(value);
+		}
+
+		public object[] FavoriteDonuts
+		{
+			get => this.Get(initialValue: Array.Empty<object>());
+			set => this.Set(value);
+		}
+
+		public bool AgreeToTermsOfServices
+		{
+			get => this.Get<bool>();
+			set => this.Set(value);
+		}
 	}
 
 	public class CreateAccountFormValidator : AbstractValidator<CreateAccountFormViewModel>
 	{
-		public CreateAccountFormValidator()
+		public CreateAccountFormValidator(IStringLocalizer localizer)
 		{
-			RuleFor(x => x.Email).NotEmpty().EmailAddress();
-			RuleFor(x => x.Password).NotEmpty();
+			var myPassword = string.Empty;
+
+			RuleFor(x => x.FirstName).NotEmpty();
+			RuleFor(x => x.LastName).NotEmpty();
+
+			RuleFor(x => x.Email)
+				.NotEmpty()
+				.EmailAddress();
+
+			RuleFor(x => x.PhoneNumber)
```

Comment:
We should count the digits, not all characters, no?

Reply:
that's true

Reply:
Ideally we would use a fixed formatting behavior like (000) 000-0000. I think this gives a good enough representation of fluent validation.

Reply:
For this sample, we could probably should remove all formatting. The important part is the data.

Reply:
Unless you really want it too be looking for digit only.

Reply:
Done